.github/workflows/php.yml

```diff
@@ -78,9 +78,6 @@ jobs:
       - name: Run phpstan
         run: ./vendor/bin/phpstan analyse -c phpstan.neon.dist
 
-      - name: Run phpstan on classes directory
-        run: ./vendor/bin/phpstan analyse -c phpstan.neon.dist -l 4 classes/
```

Comment:
Shouldn't be kept and updated to level 5 instead?

Reply:
No. The file `phpstan.neon.dist` contains the level 5. I enable `classes` in this file.

So we need to disable this specific command.